<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeystoneCornerIndicatorWidget</class>
 <widget class="QWidget" name="KeystoneCornerIndicatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <widget class="DocumentSegmentationCorner" name="topLeftCorner" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="DocumentSegmentationCorner" name="topRightCorner" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="DocumentSegmentationCorner" name="bottomLeftCorner" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="DocumentSegmentationCorner" name="bottomRightCorner" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DocumentSegmentationCorner</class>
   <extends>QWidget</extends>
   <header>document_segmentation_corner.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
